src/scores/continuous/__init__.py

```diff
@@ -44,6 +45,7 @@
     "additive_bias",
     "mean_error",
     "multiplicative_bias",
+    "additive_bias_percentage",
```

Comment:
this should be renamed pbias. After wider reading on my part, pbias is clearly the accepted name for the score, so ignore my earlier reflections on the naming. Renaming it percent_bias would also be okay, but I think pbias is what people really know it as.